integration-cli/docker_cli_run_unix_test.go

```diff
@@ -824,9 +824,30 @@ func (s *DockerSuite) TestRunTmpfsMounts(c *check.C) {
 	if _, _, err := dockerCmdWithError("run", "--tmpfs", "/run:foobar", "busybox", "touch", "/run/somefile"); err == nil {
 		c.Fatalf("/run mounted on tmpfs when it should have vailed within invalid mount option")
 	}
-	if _, _, err := dockerCmdWithError("run", "--tmpfs", "/run", "-v", "/run:/run", "busybox", "touch", "/run/somefile"); err == nil {
```

Comment:
what about the initial comment then? should we try and catch that dupication as well?